scripts.py

```diff
@@ -779,3 +784,50 @@ def __init__(self):
             SearchIndexMonitor,
             index_name=parsed.works_index,
         )
+
+class LoanReaperScript(Script):
+    """Remove expired loans and holds whose owners have not yet synced
+    with the loan providers.
+
+    This stops the library from keeping a record of the final loans and
+    holds of a patron who stopped using the circulation manager.
+
+    If a loan or (more likely) hold is removed incorrectly, it will be
+    restored the next time the patron syncs their loans feed.
```

Comment:
for my edification: is the patron's holds kept locally in SimplyE, and then merged over what the database knows about the patron's holds?